go.mod

```diff
@@ -20,5 +20,8 @@ require (
 	k8s.io/kube-openapi v0.0.0-20211110012726-3cc51fd1e909 // indirect
 	k8s.io/utils v0.0.0-20210930125809-cb0fa318a74b
 	sigs.k8s.io/cluster-api v1.0.2
+	sigs.k8s.io/cluster-api/test v1.0.2
 	sigs.k8s.io/controller-runtime v0.10.3
 )
+
+replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.0.2
```

Comment:
I had to explicitly add this since it was throwing an issue on `go mod tidy` after introducing the cluster-api/test module
```
$ go mod tidy
go: sigs.k8s.io/cluster-api/test@v1.0.2 requires
        sigs.k8s.io/cluster-api@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000
```
Not sure on why incorrect pseudo version is considered. Any idea?

Reply:
not sure, lets keep the replace directive then!